packages/rpmfile/rpmfile.0.8.0/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "RPM package reader"
+maintainer: ["Mikhail Lopatin <dx3mod@bk.ru>"]
+authors: ["Mikhail Lopatin <dx3mod@bk.ru>"]
+license: "MIT"
+tags: ["rpm"]
+homepage: "https://github.com/dx3mod/rpmfile"
+bug-reports: "https://github.com/dx3mod/rpmfile/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "angstrom" {>= "0.15"}
+  "dune" {>= "3.16"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/dx3mod/rpmfile.git"
+url {
+  src: "https://github.com/dx3mod/rpmfile/archive/refs/tags/0.8.tar.gz"
+  checksum: [
+    "md5=9cb3413fed9ea7f598106e2017ee241c"
+    "sha512=beda2fca84fa545be781a21e4763d2a0ab19ffe0819ae2710f4c7622c176a5037b4af4b76ab2e198cefcd26d12449fba1fd816c1469b7c60e4c3440f1de40533"
+  ]
+}
+x-maintenance-intent: [(latest)]
```

Comment:
that's unfortunately not the right syntax, it should be:
`x-maintenance-intent: [ "(latest)" ]`